mountinfo/mountinfo_unix.go

```diff
@@ -0,0 +1,37 @@
+// +build !windows
+
+package mountinfo
+
+import (
+	"path/filepath"
+
+	"golang.org/x/sys/unix"
+)
+
+// Mounted determines if path is mounted (i.e. it is a mount point).
+func Mounted(path string) (bool, error) {
+	// Try the fast method (of comparing devices) first.
+	// Alas, it can't detect bind mounts, so fall back on
+	// good ol' mountinfo parsing.
```

Comment:
Not sure this is the best location for this comment; thinking if it should either be;

- As part of the GoDoc comment (describing the strategy)
- Further down (before `entries, err := GetMounts(SingleEntryFilter(path))`), describing that that's where we fail back to parsing MountInfo (which may be needed if the path is a bind-mount)

Reply:
right